wordpress-mobile/WordPress-Android
# Render comments in list using same logic as comments details.

Fixes #15882

(cc @ParaskP7 )

This PR improves the way we render HTML content in Site Comment list, and removes reliance on TagSoup library in WpHtml.

I switched to using same rendering logic that is used for comment content, which means we are finally rendering mentions and images (including gifs), also HTML content, like quotes, is styled properly.

To test:
- Open the site with a variety of comments (any active p2 will do) side by side in trunk and this branch and confirm that the things look better, and nothing is broken visually.

## Regression Notes
1. Potential unintended areas of impact
 Some HTML elements might render in unexpected way.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
 No way to test html rendering at this time.

PR submission checklist:
